Makefile

```diff
@@ -0,0 +1,75 @@
+SECRET_NAME=""
+ENV_FILE=".env"
+SM2A_FOLDER="sm2a"
+
+CHDIR_SHELL := $(SHELL)
+define chdir
+   $(eval _D=$(firstword $(1) $(@D)))
+   $(info $(MAKE): cd $(_D)) $(eval SHELL = cd $(_D); $(CHDIR_SHELL))
+endef
+
+
+important_message = \
+	@echo "\033[0;31m$(1) \033[0m"
+
+info_message = \
+	@echo "\033[0;32m$(1) \033[0m"
+
+
+count_down = \
+	@echo "Spinning up the system please wait..."; \
+	secs=40 ;\
+	while [ $$secs -gt 0 ]; do \
+		printf "%d\033[0K\r" $$secs; \
+		sleep 1; \
+		: $$((secs--)); \
+	done;
+
+
+.PHONY:
+	clean
+	all
+	test
+
+all: switch-to-sm2a sm2a-local-init sm2a-local-run
+
+test:
+	$(call count_down, 10)
+
+switch-to-sm2a:
+	$(call chdir,${SM2A_FOLDER})
+
+sm2a-local-run: switch-to-sm2a sm2a-local-stop sm2a-cp-dags
```

Comment:
I assume you're running `sm2a-local-stop` first to make sure you don't try to run sm2a if it's already running?

Reply:
Yes, correct